### ISSUE TITLE
Support `arm-linux-musleabihf` and `aarch64-linux-musl` targets

### ISSUE THREAD

**Issue by j8r** [crystal-lang/crystal#5467]
`arm-linux-musleabihf` and `aarch64-linux-musl` targets will be needed to compile the crystal compiler and [package it](https://github.com/ysbaddaden/crystal-alpine/issues/6) for Alpine Linux on ARM architectures.

Looking at [Rust's targets](https://github.com/rust-lang/rust/tree/master/src/librustc_back/target) might be useful.

We should also change the triple `amd64-unknown-openbsd` to `x86_64-unknown-openbsd`

**Comment by straight-shoota** [comment]
I don't think it makes much sense to keep this issue open. It originally started with different goals and the main one has been implemented for a long time.
We've had support for `aarch64-linux-musl` since 0.25 ([#5861](https://github.com/crystal-lang/crystal/pull/5861)).

`arm-linux-musleabihf` is a different story. The focus has clearly shifted towards more modern architectures, so this is probably not that relevant.
We're open to contributions for any new target platform. But `arm-linux-musleabihf` is not a priority.

**Comment by ysbaddaden** [comment]
The `arm-linux-musleabihf` target should only be missing the `LibC` bindings since we already support `arm-linux-gnueabihf`. I can generate them if anybody needs the target.
